config/initializers/assets.rb

```diff
@@ -9,3 +9,7 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 Rails.application.config.assets.precompile += %w( search.js )
+
+# Rails.application.config.assets.precompile += %w( '.svg' )  
+# # Must include to get inline SVGs to work in deploy
```

Comment:
Maybe it'd be more illuminating for future us to move this comment above the two `Rails.application` lines and say something like `# Try uncommenting the two lines below if inline SVGs fail to display on deploy`? Right now it's somewhat confusing to say that we must include those lines if we're not actually doing so.

Reply:
you are so very right